From 0cbe77ac110d3335cafdd85196780e02d02753db Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Sat, 21 May 2022 12:37:06 +0200
Subject: [PATCH] Add <memory> include required for unique_ptr

Do no rely on <memory> to be included by some other header file.
Fixes build failures with GCC 12.1.0.

Fixes #583.
---
 include/minizinc/utils.hh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/minizinc/utils.hh b/include/minizinc/utils.hh
index 209869cb..0deedaed 100644
--- a/include/minizinc/utils.hh
+++ b/include/minizinc/utils.hh
@@ -21,6 +21,7 @@
 #include <iomanip>
 #include <iostream>
 #include <limits>
+#include <memory>
 #include <ratio>
 #include <sstream>
 #include <string>
-- 
2.36.0

